dts/nxp/s32/S32K344-172MQFP-pinctrl.h

```diff
@@ -179,6 +179,7 @@
 #define PTA16_EMIOS_0_CH11_H_I          NXP_S32_PINMUX(0, 16, 0, 59, 2)
 #define PTA17_EMIOS_0_CH6_G_O           NXP_S32_PINMUX(0, 17, 2, 0, 0)
 #define PTA17_EMIOS_0_CH6_G_I           NXP_S32_PINMUX(0, 17, 0, 54, 2)
+#define TRGMUX_INT_OUT37_EMIOS_0_CH6_G  NXP_S32_PINMUX(0, 17, 0, 54, 3)
```

Comment:
can you pls group them at the bottom of the file under `/* TRGMUX */`? this file was auto-generated with an internal script (yet to be published), let's keep the autogenerated part intact for easier maintenance. When the script is published it should also consider adding TRGMUX internal muxing for SIUL2.

Reply:
Done !